selftests/functional/test_interrupt.py

```diff
@@ -121,6 +121,9 @@ def wait_until_no_badtest():
         # Make sure the Killing test subprocess message did appear
         self.assertIn('Killing test subprocess', proc.get_output())
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
Ideally, I'd like to see this using avocado test tags.  So, we could say that those tests are time sensitive, and just do `avocado run selftests/*/*.py --filter-by-tag=-time_sensitive`, say, during RPM builds.  But this involves making those `avocado.Test`, and not `unittest.TestCase`.

What do you think?